overlays/default.nix

```diff
@@ -5,23 +5,22 @@
 
   # This one contains whatever you want to overlay
   # You can change versions, add patches, set compilation flags, anything really.
-  # https://nixos.wiki/wiki/Overlays
-  modifications = final: prev: {
-    # example = prev.example.overrideAttrs (oldAttrs: rec {
-    # ...
-    # });
-  };
+  # https://wiki.nixos.org/wiki/Overlays
+  # modifications = final: prev: {
+  #   example = prev.example.overrideAttrs (oldAttrs: rec {
+  #   ...
+  #   });
+  # };
```

Comment:
i prefer to keep this uncommented. idk why

Reply:
you so silly :3